src/diy/official_clone_trackers.md

```diff
@@ -0,0 +1,113 @@
+# Official DIY Clone Trackers
+
+Due to the nature of SlimeVR being open-source it is entirely possible to create your own trackers based on the official design files.
+Especially now that the [SlimeVR Store](https://shop.slimevr.dev) started selling components such as: cases, cables and IMUs!
+
+* TOC
+{:toc}
+
+## SlimeVR Store
+Since a little while now there is an official [SlimeVR Store](https://shop.slimevr.dev)!
+Here you can buy replacement cases, cables and even BNO modules!
+
+## Where to find the production files
+You can find the production files for both the main tracker, extension boards and the BNO module here: [Main Tracker](https://oshwlab.com/slimevr/slimevr-main-board), [Extension Board](https://oshwlab.com/eirenliel/slimevr-diy-tracker-extension) and [BNO Module](https://oshwlab.com/eirenliel/bno085-module).
+These will link you to the respective OSHWLab pages, here you can download the files or directly open them in the online version of EasyEDA.
+
+## How to order SlimeVR PCBs from a manufacturer
+Upon opening either of the two links above you will find yourself on the OSHWLab website. From here you can select: "Open in Editor" to open the files in EasyEDA.
+**You do not have to download any software for this process since EasyEDA can run in your browser!**
+This will open both the schematic and PCB Files, here you can admire, edit or adjust anything if needed for your use case (not required for official clone trackers.)
+Now head over to the top bar within easyEDA and under "Fabrication" click "PCB Fabrication file (gerber)" and click no on both options considering EasyEDA has not yet been configured and the files are already done.
+From here you can do a one click order.
+You should be able to leave all the settings as is aside from Surface Finish, set this to LeadFree HASL or ENIG!
+*You can also change the color of the solder mask if you so desire.*
+After this you should be able to continue with the ordering process and then the wait begins!
+
+**BEWARE**
+Sometimes when you get redirected to JLCPCB it won't fully load, in this case you might have to try again.
+
+## Soldering the IMU to the main PCB
+The soldering process for official slimes is a little different than your usual DIY slimes. Where normally DIY slimes are soldered together with through hole wiring, SlimeVR on the other hand uses castelated edges. This makes soldering the IMU very fast and efficient.
+
+What do you need to solder the IMU to the main PCB?
+- Soldering iron.
+- Leadfree Solder.
+- Chisel style soldering tip (optional but recommended).

```

Comment:
Could recommend specific tips used by SlimeVR?

Reply:
That is assuming they are also using a ts100 or pinecil, they are not hard to find though, so I didn't see the need for it personally.

Reply:
Indeed, or they can be used as a reference/starting point for finding similar ones. Not needed if you think it's irrelevant, just thought it might be useful :3.